lnst/RecipeCommon/Perf/Recipe.py

```diff
@@ -103,26 +118,24 @@ def time_aligned_results(self) -> "RecipeResults":
                 for measurement_iteration_result in measurement_results[i]
             ]
 
-            timestamps.append(
-                (
-                    max(
-                        [res.start_timestamp for res in iteration_results_group]
-                    ),
-                    min([res.end_timestamp for res in iteration_results_group]),
-                )
-            )
+            real_times = self._get_real_flow_times(iteration_results_group)
+            timestamps.append(real_times)
+
+        logging.info(timestamps)
```

Comment:
Should be `logging.debug()` imo. Maybe this should be completely removed or add additional message that would explain what the value is about.